motoman_mh180:
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  #kinematics_solver: lma_kinematics_plugin/LMAKinematicsPlugin
  kinematics_solver_search_resolution: 0.0050000000000000001
  kinematics_solver_timeout: 30.0
  kinematics_solver_attempts: 5
